meeting-note/meeting-note.qml

```diff
@@ -54,7 +62,12 @@ QtObject {
 
         // get the date headline
         var m = new Date();
-        var headline = headlinePrefix + " " + m.getFullYear() + "-" + ("0" + (m.getMonth()+1)).slice(-2) + "-" + ("0" + m.getDate()).slice(-2);
+
+        if (timeInNoteName) {
+          var headline = headlinePrefix + " " + m.getFullYear() + "-" + ("0" + (m.getMonth()+1)).slice(-2) + "-" + ("0" + m.getDate()).slice(-2) + "T" + ("0" + m.getHours()).slice(-2) + "." + ("0" + m.getMinutes()).slice(-2);
```

Comment:
1) do you really want to introduce a `.` in the filename?
2) you could also try to no duplicate code by just adding the time-part to the end of the currently existing headline if the flag is turned on

Reply:
Ad 1: you are right, let's keep it
Ad 2: great!

Reply:
Ad 2: Don't forget to commit your changes ;)